src/seedu/addressbook/data/person/Email.java

```diff
@@ -22,6 +22,7 @@
      * @throws IllegalValueException if given email address string is invalid.
      */
     public Email(String email, boolean isPrivate) throws IllegalValueException {
+    	super (email, isPrivate);
```

Comment:
Its good that you try super call, But you haven't provided the implementation for the constructor of Contacts. So think how you can reuse code from the parent class constructor with the super call.